tutorials/monitoring-demo/walkthrough-3.md

```diff
@@ -25,18 +25,18 @@ Processing time is the time it takes for the endpoint to process a single messag
 
 ## Sample walkthrough
 
-The following walk through uses the sample solution to simulate problems with endpoints.
+The following walkthrough uses the sample solution to simulate problems with endpoints.
 
 **Run the sample solution. Open ServicePulse to the Monitoring tab.**
 
 ![Service Pulse monitoring tab showing sample endpoints](servicepulse-monitoring-tab-sample-low-throughput.png)
 
-NServiceBus endpoints frequently rely on other resources to do their work. This might take the form of a database server that holds persisted data or a web server that hosts an API that the endpoint needs to call. The endpoints themselves are designed to tolerate failure but there are some early indicators that failure is coming.
+NServiceBus endpoints frequently rely on other resources to do their work. This might take the form of a database server that holds persisted data or a web server that hosts an API that the endpoint needs to call. The endpoints themselves are designed to tolerate failure, but there are some early indicators that failure is coming.
 
 
-### Processing messages is getting slower
+### Processing messages are getting slower
 
-The first indication that an endpoint is going to run into trouble is when processing messages starts to slow down. This is indicated by an increase in processing time. This means that database queries and web api calls are taking longer to process than they were before.
+The first indication that an endpoint is going to run into trouble is when processing messages start to slow down. This is indicated by an increase in processing time. This means that database queries and web API calls are taking longer to process than they were before.
```

Comment:
"Starts" should stay as it previously was I think.

Reply:
Correct. Good catch